### PR TITLE
RAM fixes

### PR DESCRIPTION
**Pull request by AaronDewes**
Replaces https://github.com/getumbrel/umbrel-os/pull/75

### CODE REVIEW

**Comment by mayankchhabra** [comment]
LGTM. Thanks, @AaronDewes! 

Can you please move the logic from `scripts/start` to `scripts/configure` instead as it's a one-time configuration.

File: scripts/start [hunk 1]
```diff
@@ -46,6 +46,10 @@ export DEVICE_HOST="http://"${DEVICE_IP:-"$DEVICE_HOSTNAME".local}""
 
 cd "$UMBREL_ROOT"
 
+echo "Setting dbcache size"
+DBCACHE_SIZE=$(awk '/MemTotal/{printf "%d\n", $2/2^10 * 0.5}'  /proc/meminfo)
+sed -i -e "s/dbcache=1000/dbcache=$FIXED/g" bitcoin/bitcoin.conf
```
Comment (lukechilds) [inline scripts/start:51]: Looks like you had the wrong variable name left in from development?

```suggestion
sed -i -e "s/dbcache=1000/dbcache=$DBCACHE_SIZE/g" bitcoin/bitcoin.conf
```

File: scripts/start [hunk 2]
```diff
@@ -46,6 +46,10 @@ export DEVICE_HOST="http://"${DEVICE_IP:-"$DEVICE_HOSTNAME".local}""
 
 cd "$UMBREL_ROOT"
 
+echo "Setting dbcache size"
+DBCACHE_SIZE=$(awk '/MemTotal/{printf "%d\n", $2/2^10 * 0.5}'  /proc/meminfo)
```
Comment (lukechilds) [inline scripts/start:50]: This sets `DBCACHE_SIZE` to 50% of RAM. It should be 50%-300 to account for the 300MB allocated to `maxmempool.`

```suggestion
DBCACHE_SIZE=$(awk '/MemTotal/{printf "%d\n", ($2/2^10 * 0.5) - 300}' /proc/meminfo)
```

**Comment by lukechilds** [comment]
Thanks for taking this on @AaronDewes, some tiny changes needed.

**Pull request merged**